package.json

```diff
@@ -9,8 +9,8 @@
     "npm": ">=1.3.0"
   },
   "devDependencies": {
-    "eslint": "^8.36.0",
-    "puppeteer": "^19.8.2",
+    "eslint": "^8.44.0",
+    "puppeteer": "^19.11.1",
```

Comment:
I didn't bump to `puppeteer 20.8.0` as it introduces a moderate vulnerability in dependencies:
```
word-wrap  *
Severity: moderate
word-wrap vulnerable to Regular Expression Denial of Service - https://github.com/advisories/GHSA-j8xg-fqg3-53r7
node_modules/word-wrap
  optionator  0.8.3 - 0.9.1
  Depends on vulnerable versions of word-wrap
  node_modules/escodegen/node_modules/optionator
```